core/modules/user/user.module

```diff
@@ -2503,7 +2505,7 @@ function user_role_load_by_name($role_name) {
  *
  * @param $role
  *   A role object to modify or add.
- * @return
+ * @return int
```

Comment:
```suggestion
 * @return int | bool
```

Reply:
Turns out this only ever returns an integer. If it fails, a fatal error is thrown.